### PR TITLE
[+DOC] Restore policies in restoring ILM indices

### PR DESCRIPTION
**Pull request by stefnestor**
👋 howdy! This may need Asciidoc reformatting. Will you kindly add in express commentary on [Restore a managed Datastream or Index](https://www.elastic.co/guide/en/elasticsearch/reference/master/index-lifecycle-and-snapshots.html?edit) to also restore ILM policies as needed (via `include_global_state`, [related doc](https://www.elastic.co/guide/en/elasticsearch/reference/current/create-snapshot-api.html#create-snapshot-api-request-body)). Otherwise, you induce ILM errors once ILM starts (and have to do a form of repeating the entire outlined procedure to get indices going through correctly.)

### CODE REVIEW

File: docs/reference/ilm/ilm-and-snapshots.asciidoc [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 When you restore a managed index or a data stream with managed backing indices, 
 {ilm-init} automatically resumes executing the restored indices' policies.
+As needed, make sure to <<restore-snapshot-api.html, restore>> {ilm-init} policies via `include_global_state`.
```
Comment (abdonpijpelink) [inline docs/reference/ilm/ilm-and-snapshots.asciidoc:7]:
```suggestion
To <<restore-snapshot-api,restore>> managed indices, ensure that the {ilm-init}
policies referenced by the indices exist. If necessary, you can restore
{ilm-init} policies by setting
<<restore-snapshot-api-request-body,`include_global_state`>> to `true`. 

When you restore a managed index or a data stream with managed backing indices, 
{ilm-init} automatically resumes executing the restored indices' policies.
```

**Review by abdonpijpelink — COMMENTED** [review]
Thanks for adding this note @stefnestor ! I think we should maybe open with this recommendation? I've reworded a bit, and fixed the broken link. Let me know what you think.

**Comment by stefnestor** [comment]
Sounds great. Thanks for the feedback @abdonpijpelink !

**Comment by abdonpijpelink** [comment]
@elasticmachine update branch

**Review by abdonpijpelink — APPROVED** [review]

**Pull request merged**